front/src/applications/stdcm/components/StdcmResults/StdcmResults.tsx

```diff
@@ -93,6 +98,57 @@ const StdcmResults = ({
     return extractMarkersInfo(outputs.results.simulationPathSteps);
   }, [hasSimulationResults, outputs]);
 
+  const [refSchedule, setRefSchedules] = useState<RefSchedule[]>([]);
+
+  const [postRefSchedules] = osrdEditoastApi.endpoints.postSimilarTrains.useMutation();
+  const requestTrainSchedules = async () => {
```

Comment:
You can declare this function in the useEffect directly

Reply:
yes sure